app/models/measure.rb

```diff
@@ -410,6 +410,14 @@ def quota_definition_or_nil
     end
   end
 
+  def relevant_for_country?(country_id)
+    return false if measure_excluded_geographical_areas.map(&:excluded_geographical_area).include?(country_id)
+    return true if geographical_area_id == GeographicalArea::ERGA_OMNES_ID && national?
+    return true if geographical_area_id.blank? || geographical_area_id == country_id
+
```

Comment:
I know you had had enough of this. I am happy if you refactor this in the next PR, but ideally, we should return true only once and link those two conditions with OR. I do appreciate there probably is a need for a bit of refactoring for these conditions.